.test-infra/jenkins/job_PerformanceTests_PubsubIO_Python.groovy

```diff
@@ -0,0 +1,93 @@
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+def now = new Date().format("MMddHHmmss", TimeZone.getTimeZone('UTC'))
+
+def psio_read_test = [
+        title          : 'PubsubIO Read Performance Test Python 900000 messages',
+        test           : 'apache_beam.io.gcp.pubsub_read_perf_test',
+        runner         : CommonTestProperties.Runner.DATAFLOW,
+        pipelineOptions: [
+                job_name             : 'performance-tests-psio-read-python-900000-msgs' + now,
+                project              : 'apache-beam-testing',
+                temp_location        : 'gs://temp-storage-for-perf-tests/loadtests',
+                publish_to_big_query : true,
+                metrics_dataset      : 'beam_performance',
+                metrics_table        : 'psio_read_900000msg_results',
+                // Pubsub PublishRequest can have max 10'000'000 bytes
+                input_options        : '\'{"num_records": 900000}\'',
+                num_workers          : 5,
+                autoscaling_algorithm: 'NONE',  // Disable autoscale the worker pool.
+                timeout              : 3600,
+        ]
+]
+
+def psio_write_test = [
+        title          : 'PubsubIO Write Performance Test Python 900000 messages',
+        test           : 'apache_beam.io.gcp.pubsub_write_perf_test',
+        runner         : CommonTestProperties.Runner.DATAFLOW,
+        pipelineOptions: [
+                job_name             : 'performance-tests-psio-write-python-900000-msgs' + now,
+                project              : 'apache-beam-testing',
+                temp_location        : 'gs://temp-storage-for-perf-tests/loadtests',
+                publish_to_big_query : true,
+                metrics_dataset      : 'beam_performance',
+                metrics_table        : 'psio_write_900000msg_results',
+                // Pubsub PublishRequest can have max 10'000'000 bytes
+                input_options        : '\'{' +
+                        '"num_records": 900000,' +
+                        '"key_size": 1,' +
+                        '"value_size": 1}\'',
+                num_workers          : 5,
+                autoscaling_algorithm: 'NONE',  // Disable autoscale the worker pool.
+                timeout              : 3600,
+        ]
+]
+
+def executeJob = { scope, testConfig ->
+    commonJobProperties.setTopLevelMainJobProperties(scope, 'master', 240)
+
+    loadTestsBuilder.loadTest(scope, testConfig.title, testConfig.runner,
+            CommonTestProperties.SDK.PYTHON, testConfig.pipelineOptions, testConfig.test)
```

Comment:
Recently, we've migrated Load Tests and BigQuery perf tests to Python 3.7. It would be good to keep using Python 3.7 in PubSub perf tests as well.